pkg/hintrunner/zero/zerohint_uint256.go

```diff
@@ -190,3 +190,36 @@ func createSplit64Hinter(resolver hintReferenceResolver) (hinter.Hinter, error)
 
 	return newSplit64Hint(a, low, high), nil
 }
+
+func newUint256SignedNN(a hinter.ResOperander) hinter.Hinter {
```

Comment:
Just to follow the same format as the others, the name of the function should be `new<HintName>Hint`

Reply:
corrected